tools/issue-labeler/backfill.go

```diff
@@ -58,6 +63,11 @@ func getIssues(since string) []Issue {
 		var newIssues []Issue
 		json.Unmarshal(body, &newIssues)
 		if len(newIssues) == 0 {
+			var err ErrorResponse
+			json.Unmarshal(body, &err)
+			if err.Message == "Bad credentials" {
+				glog.Exitf("Error from API: Bad credentials")
+			}
```

Comment:
Are there other errors that we could see when 0 issues are returned? We might want to at least log them just in case.